dj_rql/filter_cls.py

```diff
@@ -62,6 +64,12 @@ class RQLFilterClass:
     OPENAPI_SPECIFICATION = RQLFilterClassSpecification
     """Class for OpenAPI specifications generation."""
 
+    QUERIES_CACHE_BACKEND = LFUCache
+    """Class for query caching (can be `None`)."""
+
+    QUERIES_CACHE_SIZE = 20
```

Comment:
Just assumption -  users will want to off cache by setting QUERIES_CACHE_SIZE = 0.
LFUCache will not fail on init (LFUCache(0)). It will on 1st item set.
Maybe it's better to validate QUERIES_CACHE_SIZE before init?

Reply:
Will check for value validity or 0 at runtime to disable.